pkgs/development/libraries/cairo/default.nix

```diff
@@ -20,6 +20,14 @@ in stdenv.mkDerivation rec {
     sha256 = "0c930mk5xr2bshbdljv005j3j8zr47gqmkry3q6qgvqky6rjjysy";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Maybe add a comment with the reported issue or pr?

Reply:
@worldofpeace - Good idea; added a comment :+1: